yarn-project/sequencer-client/src/publisher/sequencer-publisher.ts

```diff
@@ -288,19 +289,29 @@ export class SequencerPublisher {
    */
   public async validateBlockForSubmission(
     header: ProposedBlockHeader,
-    attestationData: { digest: Buffer; signatures: Signature[] } = {
+    attestationData: { digest: Buffer; attestations: CommitteeAttestation[] } = {
       digest: Buffer.alloc(32),
-      signatures: [],
+      attestations: [],
     },
   ): Promise<bigint> {
     const ts = BigInt((await this.l1TxUtils.getBlock()).timestamp + this.ethereumSlotDuration);
 
-    const formattedSignatures = attestationData.signatures.map(attest => attest.toViemSignature());
-    const flags = { ignoreDA: true, ignoreSignatures: formattedSignatures.length == 0 };
+    // If we have no attestations, we still need to provide the empty attestations
+    // so that the committee is recalculated correctly
+    const ignoreSignatures = attestationData.attestations.length === 0;
```

Comment:
Ah, this is such that we could correctly figure out the proposer for the simulations. Yikes.